### ISSUE TITLE
What is the easiest way to make alternate linux distros compatible by basing it off asahi linux's kernel. 

### ISSUE THREAD

**Issue by Fyde424** [AsahiLinux/docs#130]
Are there any simple terminal scripts that could be used to do this.

**Comment by Rastafabisch** [comment]
No. You may find some already existing project which make compatible installers available. Especially well supported is [Ubuntu Asahi](https://ubuntuasahi.org), though there are many others as well. YMMV

**Comment by davide125** [comment]
Also see https://asahilinux.org/docs/SW-Alternative-Distros/
